internal/dev_server/model/store.go

```diff
@@ -15,8 +15,8 @@ const ctxKeyStore = ctxKey("model.Store")
 //go:generate go run go.uber.org/mock/mockgen -destination mocks/store.go -package mocks . Store
 
 type Store interface {
-	DeleteOverride(ctx context.Context, projectKey, flagKey string) error
-	GetDevProjects(ctx context.Context) ([]string, error)
+	DeactivateOverride(ctx context.Context, projectKey, flagKey string) error
+	GetDevProjectKeys(ctx context.Context) ([]string, error)
```

Comment:
These functions weren't really conveying what they were doing. I discovered during testing that the Delete wasn't deleting